src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -42,7 +45,10 @@ public PersonBuilder() {
         email = new Email(DEFAULT_EMAIL);
         address = Optional.of(new Address(DEFAULT_ADDRESS));
         tags = new HashSet<>();
-        moduleRoleMap = new ModuleRoleMap(new HashMap<ModuleCode, RoleType>());
+
+        HashMap<ModuleCode, RoleType> hashMap = new HashMap<>();
+        hashMap.put(DEFAULT_MODULE_CODE, DEFAULT_ROLE_TYPE);
+        moduleRoleMap = new ModuleRoleMap(hashMap);
```

Comment:
`PersonBuilder` default constructor now will create a `Person` with `CS1101S: RoleType.STUDENT` in the `moduleRoleMap` by default